tsconfig.json

```diff
@@ -4,9 +4,11 @@
     "jsx": "react",
     "types": ["node"],
     "outDir": "dist",
-    "target": "ES6",
+    "target": "ESNext",
     "allowSyntheticDefaultImports": true,
-    "moduleResolution": "NodeNext"
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "moduleResolution": "Node"
```

Comment:
These lines are redundant, they all set the same values as in the extended`tsconfig.aegir.json` - https://github.com/ipfs/aegir/blob/master/src/config/tsconfig.aegir.json#L22